x-pack/solutions/observability/plugins/streams/server/routes/streams/edit.ts

```diff
@@ -10,17 +10,20 @@ import { IScopedClusterClient } from '@kbn/core-elasticsearch-server';
 import { Logger } from '@kbn/logging';
 import { badRequest, internal, notFound } from '@hapi/boom';
 import {
+  isRootStream,
   isWiredStream,
   isWiredStreamConfig,
   streamConfigDefinitionSchema,
   StreamDefinition,
   WiredStreamConfigDefinition,
   WiredStreamDefinition,
 } from '@kbn/streams-schema';
+import fastDeepEqual from 'fast-deep-equal';
```

Comment:
Do we need a separate library for this? Why not lodash? Doesn't seem like it's on the critical path and the library looks like it's abandoned (https://github.com/epoberezkin/fast-deep-equal/issues/145)

Reply:
Sure, I can switch it to lodash. I just tried to use what other parts of Kibana were using for deeper comparisons.